### PR TITLE
Prepare libzim 7 release (Changelog, copyrights)

### LINKED ISSUE #605: Prepare release of libzim7
Changelog + other stuff.

Would like to secure python-libzim 1.0 is ready before releasing libzim7.0.0, see https://github.com/openzim/python-libzim/milestone/1

I would like as well that libkiwix is releasable before releasing libzim1.0.0, so in case there is a problem at the reader side we can quickly fix/release new versions.

### PR DESCRIPTION
**Pull request by mgautierfr**
Fixes #605

Time to finish this really long release process.

For the file copyright, I pass on each file and look in git history who modify the file. I've try to remove trivial changes.
I suppose you don't want to check all copyright on the files (it was painful). But please check your name/mail is correctly spelled
I've keep the format already existing if your copyright was somewhere else (especially if your copyrights is with a mail or not).

For authors list it was a easier, it is just a extract from git history.

Changelog is a copy a Changelog for release 6.3.2 (another branch) and full long Changelog taken from merge history.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+Version 7.0.0 is a major release.
 
-It rewrite the whole API. The changelog is not the place to
-describe the api change. A complete description of the API can be
-found in the documentation.
+The API has been completely rewritten.
+Most notable change is that namespaces are now hidden.
+The new API is described in documentation, which includes a Transition Guide from v6.
+
+ZIM files created with libzim 7 uses new ZIM major version (see Header section of spec).
```
Comment (mgautierfr) [inline ChangeLog:10]: No. It now always use the major version 6, but it was already in use with major version 5 (depending if we use
extended cluster or not). We haven't move to another major version else the backward compatibility would have been lost.

We update the minor version from 0 to 1.

Comment (rgaudin) [reply ChangeLog:10]: Ah ok, let me fix that quick

**Review by maneeshpm — APPROVED** [review]
Excited for the release!! :smile: 
Fixed some minor typos

File: ChangeLog [hunk 2]
```diff
@@ -76,7 +76,7 @@ Both backward and forward compatibility is kept.
    Illustration can now have different size and scale (even if the API do
    not use this feature) (@mgautierfr #540)
  * Add a cast to string opertor on Uuid (@maneeshpm #582)
- * Change the default cluster size to 2Mo (@mgautierfr #585)
+ * Change the default cluster size to 2Mb (@mgautierfr #585)
```
[the remark this answers is not in the document]
Comment (rgaudin) [reply ChangeLog:79]: We should keep `2MiB` in the Changelog. [See WP](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units)

**Pull request merged**